destroy_cluster.sh

```diff
@@ -35,7 +35,7 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 
-declare -r installer="${OPENSHIFT_INSTALLER:-openshift-install}"
+declare -r installer="${OPENSHIFT_INSTALLER:-$GOPATH/src/github.com/openshift/installer/bin/openshift-install}"
```

Comment:
this is a bit opinionated imho. For example, that path doesn't exist on my machine because even though I do use the "GOPATH" directory structure, I check out repositories in Git worktrees. But moreover, an `openshift-install` alias is something you've probably set to be context-independant in your shell, whereas the state of the repo may not be as predictable depending on your workflow.

I'd leave this variable as it is.

Reply:
That's a good point; I'm just following the README at the [Set Up](https://github.com/shiftstack/shiftstack-ci?tab=readme-ov-file#set-up) section, where it is recommended that you use the openshift installer from Git Hub using `go get`. This will clone the installer in the "GOPATH" directory.

Reply:
ugh we probably want to change the README then... `go get` really feels like Go v1.14

Reply:
Feel free to ignore the issue for now; there's no need to fix everything in the same PR. It's often easier to merge one thing at a time